data/agama_auto/sle_default_ipmi.jsonnet

```diff
@@ -0,0 +1,36 @@
+{
+  "product": {
+    "id": "SLES_16.0"
+  },
+  "user": {
+    "fullName": "Bernhard M. Wiedemann",
+    "password": "$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/",
+    "hashedPassword": true,
+    "userName": "bernhard"
+  },
+  "root": {
+    "password": "$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/",
+    "hashedPassword": true
+  },
+  "scripts": {
+    "pre": [
+      {
+        "name": "wipefs",
+        "body": |||
+          #!/usr/bin/env bash
+          wipefs -fa /dev/sda
```

Comment:
unattended installation can format by default, are you sure you need this?

Reply:
Yes, please see the failed job -> https://openqa.suse.de/tests/16353520 without the pre script

Reply:
see more detail info at https://suse.slack.com/archives/C082VE1U2F5/p1735877623523389

Reply:
I don't see clearly the error, but instead of the pre script there should be some syntax that we could use to delete the disk:
https://github.com/agama-project/agama/blob/master/rust/agama-lib/share/profile.schema.json#L822

Reply:
anyway good for me, we need to investigate this more.